content/en/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -23,12 +23,8 @@ compute resources for system daemons. Kubernetes recommends cluster
 administrators to configure 'Node Allocatable' based on their workload density
 on each node.
 
-## {{% heading "prerequisites" %}}
-
-{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
Please keep `{{< include "task-tutorial-prereqs.md" >}}`

Reply:
You can also drop `min-kubernetes-server-version: 1.8` from the page front matter.

Reply:
Done @sftim Back to you!